libs/components/forms/src/lib/modules/file-attachment/file-attachment/file-attachment.component.ts

```diff
@@ -277,6 +278,7 @@ export class SkyFileAttachmentComponent
     | undefined;
 
   public isImage = false;
+  public isData = false;
```

Comment:
Super minor - but can we make this `protected` instead of `public`. I know that we aren't exporting the component - but technically the variable is only be used by the template and Angular templates can now access protected variables.